CMake/External_cppzmq.cmake

```diff
@@ -0,0 +1,38 @@
+# This repo consists of a single hpp header file that wraps the ZeroMQ header.
+# If someone wants this, they probably want ZeroMQ also, however this is not a
+# dependency to "install" the cppzmq header. If ZeroMQ is not enabled, a
+# warning is shown.
+
+if(NOT fletch_ENABLE_ZeroMQ)
+  message(WARNING "cppzmq module enabled without ZeroMQ! Only the header will be installed!")
+endif()
+
+ExternalProject_Add(cppzmq
```

Comment:
This build is missing the DOWNLOAD_DIR. It is dropping the zip in the wrong directory. Also, you should use the download_name (I think that's what it is, see other packages for example) to rename the zip file to something that is recognizable as cppzmq. Right now, it's just v4.2.3.zip